pkg/apis/snapshot.storage.k8s.io/v1/types.go

```diff
@@ -0,0 +1,34 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type VolumeSnapshot struct {
```

Comment:
Couldn't get the dependencies worked out while importing. So I'm copying the real definitions for now instead of using my half-baked ones.

Reply:
Yeah I agree with nick, we should just import the types. The type should bind to the client-go/api version that we are vendoring, not the copied types though.

Reply:
The api folder should only have defied the actual crd for acorn.

Reply:
I agree that importing the types is a good idea but it causes dependency conflicts because the api types are a package of the client module. So we end up pulling in a bunch of extra imports from them. We spent a while trying to resolve the conflicts but couldn't get it working.